index.html

```diff
@@ -698,6 +669,7 @@ <h2 class="page-title">Open Source</h2>
             <div class="activity"></div>
         </div>
 </div>
+        <div class="card"></div>
```

Comment:
This does render an empty card at the end of the row. Might be nice to not have that hanging around, but I don't know if it is a show stopper.

Reply:
I suspect we will be adding more stuff here (and removing others). For example, Wonder Stuff isn't listed yet.